README.md

```diff
@@ -1,2123 +1,84 @@
-		- [Topics Object](#topicsObject)
-		- [Topic Item Object](#topicItemObject)
-		- [Stream Object](#streamObject)
-		- [Events Object](#eventsObject)
-		- [Message Object](#messageObject)
-		- [Tag Object](#tagObject)
-		- [External Documentation Object](#externalDocumentationObject)
-		- [Components Object](#componentsObject)
-		- [Reference Object](#referenceObject)
-		- [Schema Object](#schemaObject)
-		- [XML Object](#xmlObject)
-    - [Security Scheme Object](#securitySchemeObject)
-    - [Security Requirement Object](#securityRequirementObject)
-	- [Specification Extensions](#specificationExtensions)
+#### <img src="./assets/gitter.png" width="15" alt="Gitter icon">&nbsp;&nbsp; Gitter Streaming API
+Definition of the Gitter streaming API.
 
-<!-- /TOC -->
+> :point_right: [See more](./examples/gitter-streaming.yml)
 
-## Definitions
+#### :heavy_plus_sign: and more...
+Check out the [examples](https://github.com/asyncapi/asyncapi/blob/master/examples) directory for more examples.
 
-#### <a name="definitionsMessageBroker"></a>Message Broker
-A message broker is a system in charge of message exchange. It MAY provide additional features, such as message queueing, storage or processing.
+## Meetings
 
-#### <a name="definitionsMessage"></a>Message
-A message is a piece of information a process will send to a message broker. It MUST contain headers and payload.
+We meet every other Tuesday at 6:00 PM CET. Check out our [calendar](https://calendar.google.com/calendar?cid=dGJyYmZxNGRlNWJjbmd0OG9rdmV2NGxzdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) to see when we're meeting next.
 
-#### <a name="definitionstTopic"></a>Topic
-A topic is a routing key used by the message broker to deliver messages to the subscribed processes. Depending on the protocol used, a message MAY include the topic in its headers.
+Join from your favorite device: https://zoom.us/j/165106914
 
-#### <a name="definitionsProcess"></a>Process
-A process is any kind of computer program connected to a message broker. It MUST be a producer, a consumer or both.
+Or dialing:
 
-#### <a name="definitionsProducer"></a>Producer
-A producer is a process publishing messages to a message broker.
+Meeting ID: 165 106 914
 
-#### <a name="definitionsConsumer"></a>Consumer
-A consumer is a process subscribed to a message broker and consumes messages from it.
+:us: United States of America
+* [+1 720 707 2699](tel:+17207072699)
+* [+1 646 558 8656](tel:+16465588656)
 
-#### <a name="definitionsTopicTemplating"></a>Topic Templating
-Topic templating refers to the usage of curly braces ({}) to mark a section of a topic as replaceable.
+:es: Spain
+* [+34 91 198 0188](tel:+34911980188)
+* [+34 84 368 5025](tel:+34843685025)
 
-## Specification
+:gb: United Kingdom
+* [+44 203 695 0088](tel:+442036950088)
+* [+44 203 051 2874](tel:+442030512874)
 
-### Format
+Find the agenda for our past and next meetings in the [pinned issues](https://github.com/asyncapi/asyncapi/issues).
```

Comment:
I don't think such linking will scale, more and more such issues will be created I guess, I assume you want to plan one issue per meeting, because of the pinned issues limitation (3). So maybe introduce label for those `meeting` issues and then provide here a link to issues with this label

Reply:
Yeah, the idea is to just have 2 there. We then unpin one and pin a new one, but definitely labeling them is a good idea.

Reply:
yeap, but because you mention in the sentence `past` meetings, then the person that sees it, can only see one previous meeting